packages/platforms/celestica/x86-64/x86-64-cel-redstone-xp/onlp/builds/src/module/src/sfp_xfp.h

```diff
@@ -0,0 +1,294 @@
+#ifndef INCLUDE_SFP_XFP
```

Comment:
I tried to keep the changes as local as possible. so that i can move around the ONLP code just by copying the folders.